app/views/replies/preview.haml

```diff
@@ -1,3 +1,17 @@
+- if @unseen_replies.present?
+  .post-header
+    .padding-15
+      Unseen Replies
+      = link_to post_path(@post, page: 'unread', anchor: 'unread'), target: '_blank' do
+        .view-button Unread (new tab) &raquo;
+      = link_to post_path(@post, page: 'unread', anchor: 'unread') do
+        .view-button.float-left Unread &raquo;
+
+  = render partial: 'replies/single', collection: @unseen_replies, as: :reply
+  - if @unseen_replies.total_pages > 1
+    .post-ender ... and #{@unseen_replies.total_entries - @unseen_replies.per_page} more ...
```

Comment:
Also, if there are more replies and you press 'Post' again, it just reloads the page to show the rest of them. This is reasonable behavior, I guess, but could be inconvenient if people would like to post despite the other replies.

(It will also interact weirdly with people who have hidden boards from unread and are trying to still reply in them – nothing gets marked read, they can't read further in the thread, so if they respond from the last page it'll work fine but if they respond from a previous page I expect it'll go through all the ""unread"" replies. Could be fixed by pushing PRs #186 and #187.)